thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DashboardResource.java

```diff
@@ -137,18 +124,10 @@ public View getDashboardView() {
   @Produces(MediaType.APPLICATION_JSON)
   public List<String> getMetrics(@QueryParam("dataset") String collection) throws Exception {
     try {
-      collection = ThirdEyeUtils.getCollectionFromAlias(collection);
-      CollectionSchema schema = collectionSchemaCache.get(collection);
-      List<String> metrics = schema.getMetricNames();
-      CollectionConfig collectionConfig = null;
-      try {
-        collectionConfig = collectionConfigCache.get(collection);
-      } catch (InvalidCacheLoadException e) {
-        LOG.debug("No collection configs for collection {}", collection);
-      }
-      if (collectionConfig != null && collectionConfig.getDerivedMetrics() != null) {
-        metrics.addAll(collectionConfig.getDerivedMetrics().keySet());
-        metrics.removeAll(collectionConfig.getDerivedMetrics().values());
+      List<String> metrics = new ArrayList<>();
+      List<MetricConfigDTO> metricConfigs = metricConfigDAO.findActiveByDataset(collection);
+      for (MetricConfigDTO metricConfig : metricConfigs) {
+        metrics.add(metricConfig.getName());
```

Comment:
do we have alias for metric? at some point, it's better to send ID for metric and alias